uber/templates/macros.html

```diff
@@ -119,50 +119,51 @@
   </div>
 {%- endmacro %}
 
-{% macro address_form(group=False, attendee=False, is_prereg_dealer=False, copy_address=False, name_prefix="") -%}
+{% macro address_form(group=False, attendee=False, is_dealer=False, collect_full_address=False, copy_address=False, name_prefix="") -%}
     {% set group_or_attendee=group if group else attendee %}
     {% set id_prefix=group.id|replace("-","") if group else attendee.id|replace("-","") %}
-    {% if not is_prereg_dealer %}
+    {% if is_dealer or collect_full_address %}
```

Comment:
I think this conditional was the opposite of what we wanted. I was getting a bunch of javascript errors like `selectToAutocomplete() not a function` or something.